src/index.html

```diff
@@ -15,8 +15,117 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <picture class="logo-container">
+        <a
+          href="/"
+          class="logoimage"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo"
```

Comment:
Consider adding a more descriptive alt text for the logo image to improve accessibility. Instead of 'MOYO Logo', you could use something like 'MOYO Company Logo'.